CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Changed
 - Upgraded shunting-yard parent from `hotels-oss-parent` to `eg-oss-parent`.
 - Upgraded version of `hive.version` to `2.3.7` (was `2.3.4`). Allows Shunting Yard to be used on JDK>=9.
+- Cobertura removed and replaced with Jacoco for coverage.
+- Upgraded `hcommon.hive.metastore` from `1.4.1` to `1.4.2`.
+- Upgraded `circus.train` from `15.0.0` to `16.3.0`.
+- Upgraded `beeju.version` from `1.2.0` to `3.1.0`.
+- Upgraded `mockito.version` from `2.10.0` to `3.5.11`.
```

Comment:
We normally don't call out changes to our unit testing dependencies in the changelog for our apps as the end users of the apps generally don't care about these. So you can remove these 2 from there. It's different with the parent POMs as the end users of those are developers and the would care about testing library and plugin versions and changes.